extra-dev/packages/coq-libvalidsdp/coq-libvalidsdp.dev/opam

```diff
@@ -0,0 +1,54 @@
+opam-version: "2.0"
+maintainer: [
+  "Pierre Roux <pierre.roux@onera.fr>"
+  "Érik Martin-Dorel <erik.martin-dorel@irit.fr>"
+]
+
+homepage: "https://sourcesup.renater.fr/validsdp/"
+dev-repo: "git+https://github.com/validsdp/validsdp.git"
+bug-reports: "https://github.com/validsdp/validsdp/issues"
+license: "LGPL-2.1-or-later"
+
+build: [
+  ["sh" "-c" "cd libvalidsdp && ./autogen.sh && ./configure"]
+  [make "-C" "libvalidsdp" "-j%{jobs}%"]
+]
+install: [make "-C" "libvalidsdp" "install"]
+depends: [
+  "ocaml"
+  "coq" {>= "8.7" & < "8.11~"}
+  "coq-bignums"
+  "coq-flocq" {>= "3.1.0"}
+  "coq-coquelicot" {>= "3.0"}
+  "coq-interval" {>= "3.4.0" & < "4~"}
+  "coq-mathcomp-field" {>= "1.8" & < "1.10~"}
+  "ocamlfind"
```

Comment:
```suggestion
  "ocamlfind" {build}
```
@erikmd just to double check, is `ocamlfind` actually used during runtime, i.e., when the library/plugin is used inside Coq? Otherwise, I would expect the `{build}` qualification here (and also in the other file).

Reply:
@palmskog good catch, thanks! indeed the `ocamlfind` tool is not needed by the plugin at runtime (it is essentially used in the [Makefile.in](https://github.com/validsdp/validsdp/blob/896111cbd1fa866a2a3e2bc780c441c6cd20982d/Makefile.in#L37-L44))